specification/azuredependencymap/DependencyMap.Management/tspconfig.yaml

```diff
@@ -22,7 +22,6 @@ options:
     package-name: "{package-dir}"
     generate-test: true
     generate-sample: true
-    examples-dir: "{project-root}/examples"
```

Comment:
Nice catch. And you can scan current main branch to delete this config like https://github.com/Azure/azure-rest-api-specs/pull/30768.